src/definitions/modules/calendar.js

```diff
@@ -358,8 +358,9 @@ $.fn.calendar = function(parameters) {
 
               var tbody = $('<tbody/>').appendTo(table);
               i = isYear ? Math.ceil(year / 10) * 10 - 9 : isDay ? 1 - firstMonthDayColumn : 0;
+              rowsLoop:
```

Comment:
Perhaps `var` missing ?
If no `var`, the variable `rowsLoop` is accidentally exposed globally.

Reply:
Aha, this is not variable, but a label for loop !!